### DISCUSSION TITLE
Atemporal chart

### DISCUSSION THREAD

**Discussion by atilagf1** [highfestiva/finplot#396] [General]
I could use this library to generate atemporal chart candlesticks?

**Comment by highfestiva** [comment]
I don't know exactly what you mean with atemporal candlesticks, but if you mean what I think you do then yes you can. If not then maybe.

**Comment by atilagf1** [comment]
I was able to do it. But, get some problem whenI trid to use the datetime as index.

**Comment by highfestiva** [comment]
In most examples I use datetime as index, so that is probably not the direct cause of the problem.

**Reply by atilagf1** [reply]
The problem was with some indicators I used. I decide just don't use datetime for now.

**Reply by atilagf1** [reply]
I tried to correct my indicators to use with finplot with datetime as index, but it can't just zoom in and out if I use datetime. Also notice screenshot don't work also.

**Reply by highfestiva** [reply]
Example.

**Reply by atilagf1** [reply]
If I load a large time series it don't zoom. I changed my to don't use data time as index now.
